composer/workflows/airflow_db_cleanup.py

```diff
@@ -279,6 +279,10 @@ def print_configuration_function(**context):
 
 def cleanup_function(**context):
 
+    if AIRFLOW_VERSION > ['2', '2', '5']:
```

Comment:
hi all - is this a "better safe than sorry" guard? or is there genuinely something happening in >2.2.5 that will render this whole script useless?